src/navigations/TabNavigation/TabNavigation.tsx

```diff
@@ -54,7 +54,7 @@ const TabNavigation = () => {
         />
         <tab.Screen
           name={Strings.Tab_Goal}
-          component={GoalScreen}
+          component={GoalsScreenStack}
```

Comment:
Great job on solving this problem :clap: . I would urge you to right a blog on how you encountered the problem, how you researched and finally how you solved it, then share it's summary with link on socials as well.